packages/grid/x-data-grid-premium/src/tests/license.DataGridPremium.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import addYears from 'date-fns/addYears';
+import { expect } from 'chai';
+// @ts-ignore Remove once the test utils are typed
+import { createRenderer } from '@mui/monorepo/test/utils';
+import { DataGridPremium, LicenseInfo } from '@mui/x-data-grid-premium';
+import { generateLicense } from '@mui/x-license-pro';
+
+describe('<DataGridPro /> - Layout', () => {
+  const { render } = createRenderer();
+
```

Comment:
There might need the need for some clean-up logic between tests:

https://github.com/mui/mui-x/blob/645c8098b5e304726350a53f087a17eebd129d6a/packages/x-license-pro/src/useLicenseVerifier/useLicenseVerifier.test.tsx#L24-L34

Reply:
That's a tricky one
If I export `sharedLicenseStatuses`, I'm giving an easy way to totally bypass the validation by manually setting it to valid.
I can add a `LicenseInfo.reset` method and modify `useLicenseVerifier` to have it work.

Reply:
Since #6043 I am setting the license key at the before each test so we don't need to clean it up anymore.